apps/web/src/api/api.client.ts

```diff
@@ -91,3 +92,49 @@ function getHeaders() {
       }
     : {};
 }
+
+// WIP: The static API client needs to be replaced by a dynamic API client where api keys are injected.
+export function buildAPIHTTPClient({
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Prefer camel-cased acronyms for better readability‏
```suggestion
export function buildApiHttpClient({
```